tokio-fs/src/file/clone.rs

```diff
@@ -0,0 +1,33 @@
+use super::File;
+
+use futures::{Future, Poll};
+
+use std::io;
+
+/// Future returned by `File::try_clone`.
+/// Clones a file handle into two file handles.
```

Comment:
one small nit is if you could try to make this comment follow the style that we have within tokio like so https://docs.rs/tokio/0.1.14/tokio/fs/file/struct.MetadataFuture.html